qemu/tests/pci_bridge_block_stress.py

```diff
@@ -12,28 +15,27 @@ def run(test, params, env):
     1) Attach one pci-bridge to guest.
     2) Create 31 disks to this pci-bridge.
     3) Start the guest.
-    4) Check 'info block'.
-    5) Read and write data on disks under pci bridge.
+    4) Check data disks.
 
     :param test: QEMU test object
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment.
     """
 
     error_context.context("Modify params!", logging.info)
-    image_parent_bus = params.get("image_parent_bus")
-    image_num = int(params.get("image_num", 0))
-    if image_num != 0:
-        for index in range(image_num):
-            image = "stg%s" % index
-            params["images"] = ' '.join([params["images"], image])
-            params["disk_pci_bus_%s" % image] = image_parent_bus
-            params["image_name_%s" % image] = "images/%s" % image
-            params["image_size_%s" % image] = "100M"
-            params["force_create_image_%s" % image] = "yes"
-            params["remove_image_%s" % image] = "yes"
-            params["blk_extra_params_%s" % image] = "serial=TARGET_DISK%s" % index
-
+    if params["drive_format"] == "scsi-hd":
+        test.cancel("It's better to set 'driveformat' as 'virtio_blk' other than"
+                    " 'virtio_scsi' for seabios space limitation in this case.")
```

Comment:
To define `no virtio_scsi` in cfg would be better.

Reply:
Updated